package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "learn-elm",
+  "version": "0.0.1",
+  "description": "A quick intro tutorial for learning Elm",
+  "dependencies": {
+    "elm": "^0.17.1"
```

Comment:
@3tty0n _great_ question! It _will_ be by the time it's "_ready_". 😉 
(_apologies for taking so **long** to get this done; other work commitments..._)
Feel free to `git pull` on the branch and make any changes you feel appropriate and push back up on the branch/PR, I will pull before continuing my work... thanks!! ❤️